minecraft_servers/chillcenter/manifest.json

```diff
@@ -0,0 +1,34 @@
+{
+  "server_name": "chillcenter",
+  "nice_name": "ChillCenter",
+  "direct_ip": "play.chillcenter.cc",
+  "server_wildcards": [
+    "play.chillcenter.cc"
+  ],
+  "social": {
+    "web": "https://chillcenter.cc",

```

Comment:
Currently in maintenance. Update or remove.

Reply:
Removed.